src/test/java/com/github/cameltooling/lsp/internal/hover/KameletHoverTest.java

```diff
@@ -35,7 +35,7 @@
 	void testProvideDocumentationOnHover() throws Exception {
 		CamelLanguageServer camelLanguageServer = initializeLanguageServer("<from uri=\"kamelet:aws-ddb-streams-source\" xmlns=\"http://camel.apache.org/schema/spring\"></from>\n");
 		
-		HoverParams hoverParams = new HoverParams(new TextDocumentIdentifier(DUMMY_URI+".xml"), new Position(0, 21));
+		HoverParams hoverParams = new HoverParams(new TextDocumentIdentifier(DUMMY_URI+".xml"), new Position(0, 26));
```

Comment:
why changing the value? Sounds like the hover should still work on position 21

Reply:
position 21 is directly behind `kamelt:aws`. I wanted to make sure we have only one possible return value which made me change it to pos 26 which is behind `kamelt:aws-ddb`.